api/src/app.ts

```diff
@@ -1,6 +1,6 @@
 import express from 'express';
-import * as bodyParser from 'body-parser';
-import { credsRouter } from './routes/credentialsRoute';
+import bodyParser from 'body-parser'
+import { credsRouter } from './routes/credentialsRoute.js';
```

Comment:
why .js extensions for imports were added to .ts files?

Reply:
are there any implications for doing such a thing?

Reply:
Well there is likely some issue with the typescript configuration, but everything compiles down to JavaScript so you should be fine.
Don't think much side effects other than lack of consistency, I have mixed both in other projects

Reply:
ok I don't like it. lol but it works.... and I do not know an alternative.